### ISSUE TITLE
Changes made to "Repeater" fields don't appear in the admin revision history tab

### ISSUE THREAD

**Issue by happy-kaseem** [teppokoivula/VersionControl#16]
This is an issue I have noticed when working on the issue raised about restoring revisions and testing it with different types of fields. It must be only related on how the changes to "Repeater" fields are shown in the  admin revision history tab. The changes are properly recorded in the "version_control__data" and "version_control__revisions" tables and can also be restored from their.

**Comment by gerritvanaaken** [comment]
This is true for "Repeater Matrix" changes, too. My website heavily relies on Repeater Matrix sections. In facts, it’s almost all our clients can do :-) It would be perfect if they could see and revert the changes that are indeed stored in the database.
